buildenv/Build_Instructions_V11.md

```diff
@@ -219,7 +221,7 @@ The following instructions guide you through the process of building an **OpenJD
 :blue_book:
 You must install the following AIX Licensed Program Products (LPPs):
 
-- [Java9_64](https://adoptopenjdk.net/releases.html?variant=openjdk9-openj9#ppc64_aix)
+- [Java10_64](https://adoptopenjdk.net/releases.html?variant=openjdk10&jvmVariant=openj9#ppc64_aix)
```

Comment:
Why is a build of Java 10 from adopt listed under 'AIX Licensed Program Products'?

Reply:
`Java10 build from Adopt` is no longer listed under `AIX Licensed Program Products`.